### ISSUE TITLE
Palette viz should enable arrow keys for content traversal

### ISSUE THREAD

**Issue by pawn002** [pawn002/color-pair-quick-iterator#88]
As it stands right now, a screen reader user can use their screen reader hot keys to quickly navigate a table via keyboard.

_A keyboard user that is not a screen reader user cannot._

In addition, it is likely that any keyboard user would appreciate the ability to traverse the viz using arrow keys.

**Comment by pawn002** [comment]
current app has tone picker making this issue irrelevant
